src/agent/TransportService.ts

```diff
@@ -29,23 +28,24 @@ export class TransportService {
     return this.transportSessionTable[connectionId]
   }
 
-  public findEndpoint(connection: ConnectionRecord) {
+  public findServices(connection: ConnectionRecord): DidCommService[] {
```

Comment:
I'd like to be more verbose:

```suggestion
  public findDidCommServices(connection: ConnectionRecord): DidCommService[] {
```

Reply:
Good point 👍